documentation/bolt_installing_modules.md

```diff
@@ -273,44 +273,67 @@ requirement.
->   ```puppet
->     mod 'my_awesome_module', local: true
->   ```
->
->   **Bolt deletes any content in `modules/` that is not listed in your
->   Puppetfile.** If you want to keep the content, but you don't want to manage
->   it with the Puppetfile, move the content to a `site-modules` directory in
->   your project.
-
-To install a module:
-   1.  Open Puppetfile in a text editor and add the modules and versions that
-       you want to install. If the modules have dependencies, list those as
-       well. For example:
-       ```puppet
-       # Modules from the Puppet Forge.
-       mod 'puppetlabs-apache', '4.1.0'
-       mod 'puppetlabs-postgresql', '5.12.0'
-       mod 'puppetlabs-puppet_conf', '0.3.0'
-
-       # Modules from a Git repository.
-       mod 'puppetlabs-haproxy', git: 'https://github.com/puppetlabs/puppetlabs-haproxy.git', ref: 'master'
-       ```   
-   1. Run the `bolt puppetfile install` command. Bolt installs modules to the
-      first directory in the `modulepath` setting. By default, this is the
-      `modules/` subdirectory inside the Bolt project directory. To override
-      this location, update the `modulepath` setting in your [project
-      configuration file](bolt_project_reference.md).
+## Manually manage a project's modules
+
+There are a few cases where Bolt is unable to resolve module dependencies and
+manage your project's modules for you. This includes cases when you need to
+do any of the following:
+
+- Install Forge modules from an alternate Forge
+- Install git modules from a repository other than a public GitHub repository
+- Install modules through a proxy
+
+Instead of using Bolt to manage your project's modules, you will need to
+manage your project's Puppetfile manually and use Bolt to install the
+modules listed in the Puppetfile without resolving dependencies.
+
+- If you need to install modules from an alternate Forge, or install modules
+  through a proxy, set the [`module-install` configuration
+  option](bolt_project_reference.md#module-install) in your `bolt-project.yaml`
+  file. For example:
+
+  ```yaml
+  # bolt-project.yaml
+  name: myproject
+  modules: []
+  module-install:
+    forge:
+      baseurl: https://myforge.example.com
+    proxy: https://myproxy.example.com:8080
+  ```
+
+- Create a file named `Puppetfile` in your project directory. Add the
+  modules you want to install, including each module's dependencies,
+  to the Puppetfile. For example:
+
+  ```ruby
+  # Modules from an alternate Forge
+  mod 'puppetlabs/apache', '5.7.0'
+  mod 'puppetlabs/stdlib', '6.5.0'
+  mod 'puppetlabs/concat', '6.3.0'
+  mod 'puppetlabs/translate', '2.2.0'
+
+  # Modules from a private git repository
+  mod 'private-module', git: 'https://github.com/bolt-user/private-module.git', ref: 'main'
+  ```
+
+- Install the modules in the Puppetfile without resolving dependencies using
+  the following command. This command replicates the behavior of the
```

Comment:
Oh ha just had to read a little further :sweat_smile: moving this a bit higher might still be better? I defer to Heston